theblues/charmstore.py

```diff
@@ -298,7 +305,13 @@ def fetch_interfaces(self, interface, way):
                'include=charm-metadata&include=stats' +
                '&include=extra-info&include=bundle-unit-count' +
                '&limit=1000' + request)
-        data = requests.get(url, verify=self.verify)
+        try:
+            data = requests.get(url, verify=self.verify, timeout=self.timeout)
```

Comment:
I was wondering why this method didn't use `_get` - I'd be willing to make the switch if someone with more knowledge could chime in on why it isn't already.